packages/internal/metrics/src/index.ts

```diff
@@ -5,3 +5,5 @@ export {
   setPassportClientId,
   setPublishableApiKey,
 } from './details';
+export { getDetail } from './utils/state';
+export { Detail } from './utils/constants';
```

Comment:
This won't work as expected due to possible multiple instantiations of the SDK. You might get a different "details" store.